FIXME: requires libprocessui from kdebase-workspace

--- origsrc/kdevelop-3.9.97/debuggers/CMakeLists.txt	2009-12-06 20:53:21.000000000 -0600
+++ src/kdevelop-3.9.97/debuggers/CMakeLists.txt	2010-01-08 13:56:53.401249000 -0600
@@ -1,4 +1,4 @@
-if(NOT WIN32 AND NOT APPLE)
+if(NOT WIN32 AND NOT APPLE AND NOT CYGWIN)
   add_subdirectory(gdb)
-endif(NOT WIN32 AND NOT APPLE )
+endif(NOT WIN32 AND NOT APPLE AND NOT CYGWIN)
 
