--- origsrc/kdevelop-3.9.95/CMakeLists.txt	2009-08-22 17:20:03.000000000 -0500
+++ src/kdevelop-3.9.95/CMakeLists.txt	2009-12-18 04:53:17.416148200 -0600
@@ -33,7 +33,7 @@ add_subdirectory(formatters)
 add_subdirectory(languages)
 add_subdirectory(projectbuilders)
 add_subdirectory(projectmanagers)
-add_subdirectory(debuggers)
+macro_optional_add_subdirectory(debuggers)
 add_subdirectory(tools)
 add_subdirectory(xtest)
 add_subdirectory(app_templates)
