--- origsrc/kdevelop-4.1.2/languages/cpp/CMakeLists.txt	2011-01-11 14:32:11.000000000 -0600
+++ src/kdevelop-4.1.2/languages/cpp/CMakeLists.txt	2011-01-14 05:07:07.222700600 -0600
@@ -15,7 +15,7 @@ if(CMAKE_COMPILER_IS_GNUCXX)
   macro_ensure_version("4.3.0" "${_gcc_version}" GCC_IS_NEWER_THAN_4_3)
   if (GCC_IS_NEWER_THAN_4_3)
     message(STATUS "Enabling c++0x support for unordered map")
-    add_definitions( -std=c++0x ) # For unordered_map
+    add_definitions( -std=gnu++0x ) # For unordered_map
   else(GCC_IS_NEWER_THAN_4_3)
     add_definitions( -DGXX_LT_4_3 )
   endif (GCC_IS_NEWER_THAN_4_3)
