--- origsrc/kdevelop-3.91/languages/cpp/cppduchain/declarationbuilder.cpp	2008-05-27 07:25:38.000000000 -0500
+++ src/kdevelop-3.91/languages/cpp/cppduchain/declarationbuilder.cpp	2008-07-29 00:17:51.718750000 -0500
@@ -493,8 +493,8 @@
       declaration = new NamespaceAliasDeclaration(m_editor->currentUrl(), newRange, scope, currentContext());
       declaration->setSmartRange(range);
       declaration->setIdentifier(customName);
-    } else if (isForward) {
-      declaration = specialDeclaration<ForwardDeclaration>(range, newRange, scope);
+//    } else if (isForward) {
+//      declaration = specialDeclaration<ForwardDeclaration>(range, newRange, scope);
 
     } else if (isFunction) {
       if (scope == Declaration::ClassScope) {
